### DISCUSSION TITLE
Share files with others Apps (share sheet)

### DISCUSSION THREAD

**Discussion by edugoia** [flet-dev/flet#3292] [Extensions]
For more robust application development, a function could be introduced that calls the native sharing menu "share sheet"

**Comment by ndonkoHenri** [comment]
Whats "sheet" in this case?

**Reply by edugoia** [reply]
It is the name of the native menu on Android, https://developer.android.com/training/sharing/send?hl=pt-br

![sharesheet](https://github.com/flet-dev/flet/assets/169856714/87590d91-764c-4e49-93b3-061d1b563799)

**Reply by Muddassir-Farooq-official** [reply]
@ndonkoHenri waiting for this functionality

**Comment by GreenDringo42** [comment]
I'm still waiting for this
